lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -129,8 +129,10 @@ def correction(node)
         def frozen_strings?
           return true if frozen_string_literals_enabled?
 
-          frozen_string_cop_enabled = config.for_cop('Style/FrozenStringLiteral')['Enabled']
```

Comment:
Typo since introduction, was always true for most users since `Enabled` came from the department.